src/Lib/CliArgsProcessor.php

```diff
@@ -79,7 +79,10 @@ private static function help(): string {
 			. "Examples:\n"
 			. "$script --pid=100 --pid-file=/var/run/manticore/searchd.pid\n"
 			. "$script --pid=100 --pid-file=/var/run/manticore/searchd.pid --host=127.0.0.2 --port=1000\n"
-			. "$script --pid=100 --pid-file=/var/run/manticore/searchd.pid --disable-telemetry\n\n";
+			. "$script --pid=100 --pid-file=/var/run/manticore/searchd.pid --disable-telemetry\n\n"
+			. "Detailed info on Manticore executor can be found "
+			. "\e]8;;https://github.com/manticoresoftware/executor/\e\\here\e]8;;\e\\ "
```

Comment:
I think it's okay just leave a link here or maybe even better just add it to README. Actually, this tool will be used only for development purposes, eg – contributors, so it makes sense to keep this info in the README file only and remove it from the code.